etc/fuel_config.json

```diff
@@ -1,82 +0,0 @@
-{
-    "flask": {
-        "PORT": 5000,
-        "HOST": "0.0.0.0",
-        "DEBUG": true
-    },
-
-    "global": {
-        "title": "Fuel DevOps Portal",
-        "logo_txt": "DevOps Portal",
-        "name": "Fuel DevOps",
-        "logo": "fuel-logo.png",
-        "css": "fuel-main.css"
-    },
-
-    "cloud_status": {
-        "enabled":  true,
-        "enabled_dashboards": ["availability", "health"],
-        "regions": [
-            {
-                "region": "west-1.hooli.net",
-                "health": {
-                    "elastic": "<elastic_url>/<elastic_index>/<elastic_type>"
-                },
-                "availability": {
-                    "graphite": "<graphite_connection_string>",
-                    "services": {
-                        "nova": "<put_line_here>",
-                        "glance": "put_line_here"
-                    }
-                }
-            }
-        ]
-    },
-    "infra": {
```

Comment:
Standartization is good, but we need to have this information specified from configuration file and to have api that returns available infra resources, so we do need to keep this